src/material-experimental/mdc-list/list-option.scss

```diff
@@ -18,6 +19,12 @@
     @include mdc-checkbox.without-ripple($query: animation);
   }
 
+  .mdc-checkbox {
```

Comment:
Is there a way we can somehow share the same styles with mat-checkbox so we don't wind up with another whole copy?

Reply:
We can't use the MDC checkbox here directly, because the checkbox here is decorative. It would also increase the bundle size, because it'll bring in MDC's JS. We could use the non-MDC pseudo checkbox, but it won't look exactly like the MDC version. Eventually we could build a pseudo checkbox based on MDC, but last time I checked some of the selectors looked something like `input:checked ~ .checkmark` which won't work without the `input`.

Reply:
Can you just add a comment explaining that